common/persistence/dataInterfaces.go

```diff
@@ -467,7 +466,7 @@ type (
 		ExecutionStats      *persistenceblobs.ExecutionStats
 		BufferedEvents      []*historypb.HistoryEvent
 		VersionHistories    *VersionHistories
-		Checksum            checksum.Checksum
+		Checksum            persistenceblobs.Checksum
```

Comment:
After merging with WEI, nothing will block from converting entire WMS to proto.

Reply:
Will the checksum value stay the same? If not, will this break backward compatibility?

Reply:
Yes, proto struct was already used for persistence. But before they were converted (basic 1:1 mapping) to go struct, go struct was used in code, and then converted back to proto for persistence. This PR just removes this mapping and change all internal code to use proto struct, not go struct.

Reply:
They way how we compute crc32 stays the same.